### DISCUSSION TITLE
[Bug] Errors doesn't work with setValue and resolver

### DISCUSSION THREAD

**Discussion by a1exymoroz** [react-hook-form/react-hook-form#6539] [Q&A]
Hi guys, I have a question about errors. Why don't errors change when we have the resolver for a custom logic?

To Reproduce

1. Go to https://codesandbox.io/s/react-hook-form-v7-seterror-forked-filrz?file=/src/index.jsx
2. Change first and second numbers
3. Second value must be greater than first value
4. An error is displayed when changing the first value.
5. the  error isn't hidden when changing the second value

![react-hook-form](https://user-images.githubusercontent.com/28485877/133142348-fec2affe-550f-4e38-b60a-b2a01cc4e37e.gif)

**Comment by a1exymoroz** [comment]
up

**Comment by bluebill1049** [comment]
hook form optimise error for per input, i think this RFC is probably what you after: https://github.com/react-hook-form/react-hook-form/discussions/6472
